test/unitTests/features/reportIssue.test.ts

```diff
@@ -37,19 +36,29 @@ suite(`${reportIssue.name}`, () => {
             version: "version2",
             isBuiltin: false
         },
-        id: "id2"
+        id: "id2",
+        extensionPath: "c:/extensions/xyz-x64"
+    };
+
+    const fakeDotnetInfo: DotnetInfo = {
+        FullInfo: "myDotnetInfo",
+        Version: "1.0.x",
+        RuntimeId: "1.1.x"
     };
 
     let fakeMonoResolver: FakeMonoResolver;
     let fakeDotnetResolver : FakeDotnetResolver;
-    let eventStream: EventStream;
-    let eventBus: TestEventBus;
-    let getDotnetInfo = FakeGetDotnetInfo;
+    let getDotnetInfo = async () => Promise.resolve(fakeDotnetInfo);
     let options: Options;
     let issueBody: string;
 
     setup(() => {
         vscode = getFakeVsCode();
+        // @ts-ignore
```

Comment:
It needs to return a vscode.Uri, but the only way I know how to get one is to call vscode.Uri.Parse... which we're implementing here.  Since the result of this call doesn't matter for the unit test anyway, I return the same input value.  Since it isn't a vscode.Uri type though I need to suppress the TS warnings.

When we get integration tests I want to delete these unit tests in favor of integration tests anyway.

Reply:
@dibarbet The VSCode URI package is referencable separately: https://github.com/microsoft/vscode-uri. So it's pretty easy to add a reference to that and then have a "real" parse method in play.